src/cms/tests/views/api_view_test.py

```diff
@@ -26,6 +28,11 @@ class APIViewTest(ViewTest):
 
     fixtures = ["src/cms/fixtures/test_data.json"]
 
+    @classmethod
+    def setUpClass(cls):
+        with disable_listeners():
+            super().setUpClass()
+
```

Comment:
Very cool PR :rocket: 
I'm really tempted to wait for its merge
Only question is: @ulliholtgrave  when this week the UI tests take place?

Reply:
The tests will start on Wednesday, I guess we can't wait for the merge. Is this is a critical bug or are we able to merge it and provide a patch afterwards?

Reply:
And even if I apply @timoludwig first workaround option (e14762f7cb7a0c49a917efccd39454ca8c48c29f), loading the fixtures still fails:
![Screenshot from 2021-05-10 12-03-35](https://user-images.githubusercontent.com/47010475/117642786-00adf080-b188-11eb-9bb5-7c23d5b5e0db.png)
Not because the `get_absolute_url()` in our model `PageTranslation`, but one [line later](https://github.com/DjangoAdminHackers/django-linkcheck/blob/d086a1ef2d8db56c6fe3f4e49f80238b9bcce782/linkcheck/listeners.py#L119), which should be fixed by Timo's PR, so then only the settings file option is left?

Reply:
But maybe, we don't need another settings file for it...
If we put
```
if "LINKCHECK_DISABLE_LISTENERS" in os.environ:
    LINKCHECK_DISABLE_LISTENERS = True
```
in the normal settings file and put
```
export LINKCHECK_DISABLE_LISTENERS=1
```
in the loaddata script, maybe that works... :thinking: